musicbot/bot.py

```diff
@@ -1464,15 +1464,26 @@ def check(m):
     async def cmd_volume(self, message, player, new_volume=None):
         """
         Usage:
-            {command_prefix}volume (+/-)[volume]
+            {command_prefix}volume
+
+        Shows the current volume.
+        """
+
+        if not new_volume:
+            return Response('Current volume: `%s%%`' % int(player.volume * 100), reply=True)
+			
+    async def cmd_setvolume(self, message, player, permissions, new_volume=None):
```

Comment:
This shouldn't be it's own command. For efficiency, it should be a part of the `volume` command, like it has always been.